File: src/db/models/user_settings.js
"use strict";

module.exports = (sequelize, DataTypes) => {
  const user_settings = sequelize.define(
    "user_settings",
    {
      user_id: DataTypes.STRING,
      settings: DataTypes.JSONB,
      version: DataTypes.INTEGER
    },
    {}
  );
  return user_settings;
};

File: src/db/models/user_settings.py
from sqlalchemy import Column, String, Integer, DateTime
from sqlalchemy.dialects.postgresql import JSONB
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

class UserSettings(Base):
    __tablename__ = "user_settings"

    user_id = Column(String, primary_key=True)
    settings = Column(JSONB)
    version = Column(Integer)
    createdAt = Column(DateTime)
